codes/python/chapter_array_and_linkedlist/array.py

```diff
@@ -4,22 +4,27 @@
 Author: Krahets (krahets@163.com)
 """
 
-import sys, os.path as osp
+import os.path as osp
+import random
+import sys
+from typing import List
```

Comment:
I'd recommend using `from typing import List` to simplify the codes. This behavior is similar to the LeetCode editor.